lib/themes/dosomething/paraneue_dosomething/js/lab.js

```diff
@@ -0,0 +1,24 @@
+import Onboarding from './components/Onboarding';
+import ContextSlide from './components/ContextSlide';
+
+const $ = require('jquery');
+const React = require('react');
+const ReactDom = require('react-dom');
```

Comment:
Hmm, I totes get making this camel-cased for consistency but it looks so weird. I could get used to it. 💭